views/js/controller/inspectResults.js

```diff
@@ -105,6 +106,24 @@ define([
                     taskCreationData : {uri : uniqueValue}
                 }).start();
             });
+
+            binder.register('open_url', function(actionContext) {
+                const data = _.pick(actionContext, ['uri', 'classUri', 'id']);
```

Comment:
Since you aren't doing anything with this data on the backend, no need to send it?

Reply:
BE needs this data in request body to send the right proctorio review link